### PR TITLE
support terragrunt

### PR DESCRIPTION
**Pull request by Spartakovic**

### CODE REVIEW

File: pkg/terraform/tf.go [hunk 1]
```diff
@@ -1,24 +1,55 @@
...
+	return terragrunt.runTerragruntCommand("apply")
+}
+
+func (terragrunt Terragrunt) Plan() (bool, string, string, error) {
+	stdout, stderr, err := terragrunt.runTerragruntCommand("plan")
+	return true, stdout, stderr, err
+}
+
+func (terragrunt Terragrunt) runTerragruntCommand(command string) (string, string, error) {
+	cmd := exec.Command("terragrunt", command, "--terragrunt-working-dir", terragrunt.WorkingDir)
```
Comment (motatoes) [inline pkg/terraform/tf.go:37]: is terragrunt executable available in the github action by default?

Comment (Spartakovic) [reply pkg/terraform/tf.go:37]: nope, I assumed users will install it with some other action

Comment (Spartakovic) [reply pkg/terraform/tf.go:37]: should we install it?

Comment (motatoes) [reply pkg/terraform/tf.go:37]: You're right, I think it makes sense to install prior step, lets merge it and test it out

**Review by motatoes — APPROVED** [review]

**Pull request merged**